From: Ran Xiaokai <ran.xiaokai@zte.com.cn>
Subject: mm/page_alloc: update comments for rmqueue()
Date: Sun, 18 Sep 2022 02:56:40 +0000

Since commit 44042b449872 ("mm/page_alloc: allow high-order pages to be
stored on the per-cpu lists"), the per-cpu page allocators (PCP) is not
only for order-0 pages.  Update the comments.

Link: https://lkml.kernel.org/r/20220918025640.208586-1-ran.xiaokai@zte.com.cn
Signed-off-by: Ran Xiaokai <ran.xiaokai@zte.com.cn>
Cc: Mel Gorman <mgorman@techsingularity.net>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---


--- a/mm/page_alloc.c~mm-page_alloc-update-comments-for-rmqueue
+++ a/mm/page_alloc.c
@@ -3810,7 +3810,8 @@ static struct page *rmqueue_pcplist(stru
 }
 
 /*
- * Allocate a page from the given zone. Use pcplists for order-0 allocations.
+ * Allocate a page from the given zone.
+ * Use pcplists for THP or "cheap" high-order allocations.
  */
 
 /*
_
